charts/sonarqube/templates/config.yaml

```diff
@@ -8,18 +8,15 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  {{- if and .Values.sonarSecretKey (not .Values.sonarProperties) (not .Values.elasticsearch.bootstrapChecks) }}
-  sonar.properties: sonar.secretKeyPath={{ .Values.sonarqubeFolder }}/secret/sonar-secret.txt
-  {{- end }}
-  {{- if or .Values.sonarProperties (not .Values.elasticsearch.bootstrapChecks) }}
+{{- if or .Values.sonarProperties .Values.sonarSecretKey (not .Values.elasticsearch.bootstrapChecks ) }}
```

Comment:
This condition is superfluous for me. In the deployment and stateful set, we only get data from this configmap if this condition is met. So, we do a double check that is not needed. However, for readability purposes, I see the opportunity of leaving this condition here, as it improves our understanding of the configmap (maybe a third opinion from @leo-geoffroy-sonarsource is also needed here 😝).